zipline-cli/src/test/kotlin/app/cash/zipline/cli/DownloadTest.kt

```diff
@@ -89,6 +89,8 @@ class DownloadTest {
     // Zipline files
     val applicationName = "app1"
     val manifest = ZiplineManifest.create(
+      applicationId = "",
```

Comment:
should we make these nullable? Empty is kind of the worst of both worlds; it’s not usable but it looks usable

Reply:
In honesty, I was lazy in the first pass of filling testing values and was planning to come back to it in a follow up commit. As an API, it probably should be non-null so there isn't ambiguity on whether to include it in the manifest or not.